packages/deployer/tasks/deploy.js

```diff
@@ -15,6 +15,7 @@ const {
 const logger = require('../utils/logger');
 const prompter = require('../utils/prompter');
 const relativePath = require('../utils/relative-path');
+const { capitalize } = require('../utils/string');
 
 const isWord = (str) => /^[\w\d]+$/.test(str);
```

Comment:
np (or future improvement). Could we move those definitions (and other regex) to the string utils?